templates/cisco_wlc_ssh_show_advanced_802.11a_channel.template

```diff
@@ -0,0 +1,50 @@
+Value ASSIGN_MODE (\w+)
+Value UPDATE_TIME (\d+ \w+)
+Value ANCHOR_TIME (\d+)
+Value NOISE (\w+)
+Value INTERFER (\w+)
+Value LOAD (\w+)
+Value DEVICE_AWARE (\w+)
+Value CLEANAIR_DRIVEN (\w+)
+Value LEADER (\w+)
+Value DCA_SENSITIVITY (\w+ \(\d+ dB\))
+Value DCA_WIDTH (\d+ \w+)
+Value DCA_MIN (-\d+ dBm)
+Value List DCA_CHANNELS ((\d+,?)*?)
+
+Start
+  ^\s*Leader\sAutomatic\sChannel\sAssignments*$$
+  ^\s+Channel\sAssignment\sMode\.*\s${ASSIGN_MODE}s*$$
+  ^\s+Channel\sUpdate\sInterval\.*\s${UPDATE_TIME}s*$$
+  ^\s+Anchor\stime\s\(Hour\sof\sthe day\)\.*\s+${ANCHOR_TIME}\s+s*$$
+  ^\s+Update\sContributions*$$
+  ^\s+Noise\.*\s${NOISE}s*$$
+  ^\s+Interference\.*\s${INTERFER}s*$$
+  ^\s+Load\.*\s${LOAD}s*$$
+  ^\s+Device\sAware\.*\s${DEVICE_AWARE}s*$$
+  ^\s+CleanAir\sEvent-driven\sRRM option\.*\s${CLEANAIR_DRIVEN}s*$$
+  ^\s+Channel\sAssignment\sLeader\.*\s+${LEADER}\s+\(([\d1-9]+\.?){4}\)\s+\(\:\:\)s*$$
+  ^\s+Last\s+Run\.*\s+\d+\s+seconds\s+agos*$$
+  ^\s+Last\s+Run\s+Time\.*\s+\d+\s+secondss*$$
+  ^\s+DCA\sSensitivity\sLevel\:?\s?\.*\s${DCA_SENSITIVITY}s*$$
+  ^\s+DCA\s\S+\sChannel\sWidth\.*\s${DCA_WIDTH}s*$$
+  ^\s+DCA\sMinimum\sEnergy\sLimit\.*\s${DCA_MIN}s*$$
+  ^\s+Channel\sEnergy\sLevels
+  ^\s+Minimum
+  ^\s+Average
+  ^\s+Maximum
+  ^\s+Channel\sDwell\sTimes
+  ^\s+802\.11a\s5\sGHz\sAuto\-RF Channel Lists*$$
+  ^.*Allowed\sChannel\sList\.*\s${DCA_CHANNELS}s*$$ -> Channels
```

Comment:
This attempts to capture allowed channels on both 5 and 4.9Ghz.

I would say that we'd want to capture those in separate variables and then post process them in needed.

This would be to do state transitions on L37 instead of L38 and separate variables.